packages/components/NativeRadioButton/src/NativeRadioButton.tsx

```diff
@@ -0,0 +1,70 @@
+/** @jsx withSlots */
+import { ViewProps } from 'react-native';
+import { compose, buildProps, mergeProps, UseSlots, withSlots, LayoutTokens } from '@fluentui-react-native/framework';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const nativeRadioButtonName = 'NativeRadioButton';
+
+const NativeRadioButtonView = ensureNativeComponent('RadioButtonView');
+
+export type NativeRadioButtonProps = {
+  /**
+   * Radio button title, default title is set to "Button".
+   */
+  title?: string;
+  /**
+   * Whether the radio button is selectable or not, true by default.
+   */
+  enabled?: boolean;
+  /**
+   * Current state of the radio button. There are two states:
+   * true -> On
+   * false -> Off
+   * false by default.
+   */
+  state?: boolean;
+  /**
+   * Unique identifier for each radio button.
+   */
+  buttonKey?: string;
+  /**
+   * On press event block
+   */
+  onPress?: () => void;
+};
+
+export type NativeRadioButtonTokens = LayoutTokens;
+
+export type NativeRadioButtonViewProps = NativeRadioButtonProps & NativeRadioButtonTokens & ViewProps;
+
+interface NativeRadioButtonType {
+  props: NativeRadioButtonViewProps;
+  slotProps: { root: NativeRadioButtonViewProps };
+  tokens: NativeRadioButtonTokens;
+}
+
+export const NativeRadioButton = compose<NativeRadioButtonType>({
+  displayName: nativeRadioButtonName,
+  tokens: [
+    {
+      minWidth: 100,
+      minHeight: 20,
+    },
+    nativeRadioButtonName,
+  ],
+  slots: { root: NativeRadioButtonView },
+  slotProps: {
+    root: buildProps((tokens) => ({
+      style: {
+        marginLeft: 3,
+        marginTop: 3,
+        width: tokens.minWidth,
+        height: tokens.minHeight,
```

Comment:
style has a `minWidth` and `minHeight` key. I think those are better suited for these tokens.

Reply:
But `width` and `height` must be defined to get the native views showing in the app?

Reply:
I think you can just set minHeight/minWidth and it'll display too. You might have an issue where the control grows too big and takes up more space then you wanted it to (I did). Id that's the case, then yeah I guess we can just use width/Height.

An aside, I was wondering why out native modules need a hardcoded size and I came across this StackOverflow. The solution to get our components to use something akin to `intrinsicContentSize` might be to implement a shadow view for each component https://stackoverflow.com/a/59746747 Definitely out of scope and something I wanna investigate myself later

Reply:
minHeight & minWidth worked, thanks for the suggestion.